### DISCUSSION TITLE
DBA / Librarian -  I want to document opera performances and productions so that information is available to the industry and historians/researchers.

### DISCUSSION THREAD

**Discussion by mariel-marshall** [pair-cg/general-contributions#26] [User Stories]
### Original User Story: 
“As a DBA/Librarian and opera stage manager I want to document opera performances and productions so that information is available to the industry and historians/researchers.”

# 1. Need:
- preserve information about specific opera performances and productions in electronic format 
     - preserve structure of work (number of characters, voice types, descriptions of characters, size of ensemble) and individuals involved in production as well as performance
- Ensure that the information is available electronically, upon request  - without human intervention
-  bring attention to new works (to be created)
- bring attention to new creators

# 2. Action:
_What is an action that can be taken towards fulfilling that need?_
- Store information in an information system
- Store information in a shoe box
- Service external information requests manually (fax, email, post)
- Service external information requests automatically
- Store in Rob Hudson Performance Database

# 3. Desired Outcome:
_What is the desired outcome?_

- Complete documentation of specific opera performances and productions
- Seekers of information are aware of the information in the database
- Seekers of information are served information in a way that they can process.

# 4. Perspective:
_From what perspective is this user story coming from?_
Executive Director of an arts organization

# 5. Stakeholder Group:
Opera Industry members (producers?)
Non-operatic Industry Members 
Archivist / Librarian 

# 6. Additional info:
_Here's more information._
Need Opera v. Chamber work definition
- industry

Possible question: what opera is there for # of x voice types 
- children operas

**Comment by samipierre** [comment]
**Question from PAIR-CG monthly meeting:**

Do we need sub user stories for each type of data user? [industry (economic) v. historian]

**Answer**: 

User stories are in the perspective of one person/organization. So as an archivist I provide data I have one user story and can list who I think the data may be useful to.
If they are consuming the data then yes, but as an archivist you do not need to know how others will use your data. 

----------

Consider: desired out come is that other people can access this other data.

**Reply by joneswwilson** [reply]
Action:  Make data available via LOD similar to the Carnegie Hall Performance database (http://data.carnegiehall.org/sparql/).  Provide details of performances consisting of data like that in the Met Opera's database (http://archives.metoperafamily.org/archives/frame.htm).

**Reply by joneswwilson** [reply]
Perspective: 
avid opera enthusiast- learning about past performances and performers;
researcher- information about works performed, performers, creators within specific time periods;
industry administrator- identify works with specific characteristics (voices, instrumental ensemble size, etc.), creators, performance history.

**Reply by joneswwilson** [reply]
Additional information- using LOD allows making connections among multiple performing arts presenters, e.g. identifying people who performed at Carnegie Hall, the MET, NY Philharmonic, Chicago Lyric Opera, BAM, and in different genres.

**Comment by samipierre** [comment]
### Few questions for you 
1. Is there something preventing you from currently documenting and preserving this information? 
2. What issues are you having documenting the structure of the work? 
3. Are you struggling with how to provide this structure to consumers? 
4. What types of requests/queries are you getting for your database?
5. Also why are you interested in using the Carnegie Hall Performance database schema?

**Reply by joneswwilson** [reply]
1. Is there something preventing you from currently documenting and preserving this information?

No.  In fact I have an MS access database, which is now being migrated to a cloud-based MySQL db, with data for my organization.

However, it is specific to one organization, using a custom structure, and not available to anyone else.

2. What issues are you having documenting the structure of the work?

The primary issue is whether to use existing vocabularies or creating a new one, more familiar to the opera industry.

3. Are you struggling with how to provide this structure to consumers?

The approach used by Carnegie Hall to contribute their LOD for the 
Performance History db to Wikidata is the model I'm considering.

4. What types of requests/queries are you getting for your database?

Occasional request from another company to recommend artists we have engaged.

5. Also why are you interested in using the Carnegie Hall Performance database schema?

Although that schema works for the CH Performance History db, I have developed a model which allows the capture of more details which are specific to the opera genre.
